src/jobs/stalebot.ts

```diff
@@ -107,34 +107,58 @@ const closeStalePullRequests = async (
   stalePullRequests,
   now: moment.Moment
 ) => {
-  await Promise.all(
-    stalePullRequests.map((pullRequest) => {
-      if (now.diff(pullRequest.updated_at, 'days') >= DAYS_BEFORE_CLOSE) {
-        return org.api.issues.update({
-          owner: org.slug,
-          repo: repo,
-          issue_number: pullRequest.number,
-          state_reason: 'not_planned',
-          state: 'closed',
-        });
-      }
-      // If issue has been updated in the last day, we know that there has been activity
-      else if (now.diff(pullRequest.updated_at, 'hours') <= HOURS_IN_DAY) {
+  const stalePullRequestUpdates = stalePullRequests.map(async (pullRequest) => {
+    // Check if there was non-bot activity after we last labeled it stale
+    const { data: events } = await org.api.issues.listEvents({
+      owner: org.slug,
+      repo: repo,
+      issue_number: pullRequest.number,
+      per_page: 100,
+    });
+
+    const staleLabelEvent = events
+      .reverse()
+      .find(
+        (event) =>
+          event.event === 'labeled' && event.label?.name === STALE_LABEL
+      );
+
+    if (staleLabelEvent) {
+      const staleLabeledAt = moment(staleLabelEvent.created_at);
+      const hasActivityAfterStale = events.some(
+        (event) =>
+          moment(event.created_at).isAfter(staleLabeledAt) &&
+          event.actor?.id !== GETSENTRY_BOT_ID
```

Comment:
I don't think the constant here is accurate when it comes to actor, it is `66042841` if I look type in

```
gh api repos/getsentry/sentry/issues/76411/timeline
```